test/integration/etcd_storage_path_test.go

```diff
@@ -1028,21 +835,15 @@ func TestEtcd3StoragePath(t *testing.T) {
 	// use the loopback config because it identifies as having the group system:masters which is a "magic" do anything group
 	// for upstream kube.
 	kubeConfigFile := masterConfig.MasterClients.OpenShiftLoopbackKubeConfig
+	kubeConfig := testutil.GetClusterAdminClientConfigOrDie(kubeConfigFile)
+	kubeConfig.QPS = 99999
```

Comment:
5 9's :)

Reply:
> 5 9's :)

should be big enough for anyone.  On my machine took the test from three minutes to 45 seconds.  And I got tired of waiting.